stream_alert/alert_processor/outputs.py

```diff
@@ -364,6 +365,32 @@ def _item_verify(self, item_url, item_str, item_key, item_type):
             'type': item_type
         }
 
+    def _incident_assignment(self, context):
+        """Method to determine if the incident gets assigned to a user or an escalation policy
+
+        Args:
+            context (dict): Context provided in the alert record
+
+        Returns:
+            tuple: assigned_key (str), assigned_value (dict to assign incident to an escalation
+            policy or array of dicts to assign incident to users)
+        """
+        # Check if a user to assign the incident is provided
+        user_to_assign = context.get('assigned_user', False)
+
+        # If provided, verify the user and get the id from API
+        if user_to_assign:
+            user_assignee = self._user_verify(user_to_assign)
+            # User is verified, return tuple
+            if user_assignee:
+                return 'assignments', [{'assignee': user_assignee}]
+
+        # If escalation policy was not provided, use default one
+        policy_to_assign = context.get('assigned_policy') or self._escalation_policy
```

Comment:
with a lookup like this, you can use a default value with the get like `context.get('assigned_policy', self._escalation_policy)` instead of doing the `... or ...`